mlir/lib/Dialect/Func/Utils/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_mlir_dialect_library(MLIRFuncUtils
+  Utils.cpp
+
+  ADDITIONAL_HEADER_DIRS
+  ${MLIR_MAIN_INCLUDE_DIR}/mlir/Dialect/Func/Utils
+
+  LINK_LIBS PUBLIC
+  MLIRFuncDialect
+  MLIRComplexDialect
```

Comment:
Why do we need this?

Reply:
You shouldn't need the complex dialect for that. You'll have to debug your linking, it does not make sense to have a dependency from FuncUtils on the complex dialect.